asimov

```diff
@@ -49,6 +49,26 @@ readonly ASIMOV_VENDOR_DIR_SENTINELS=(
     'vendor Gemfile'              # Bundler (Ruby)
 )
 
+# A list of "fixed directories".
+#
+# Directories will be always excluded
+readonly ASIMOV_FIXED_DIRS=(
+  ~/.cache
+  ~/.dts
+  ~/.kube/cache
+  ~/.kube/http-cache
+  ~/.m2
```

Comment:
I would suggest to don't exclude ~/.m2, but  ~/.m2/repository.
settings.xml is inside ~/.m2 and is very important file (can keep passwords and secrets)